### ISSUE TITLE
Consider dispatching loadstart "synchronously" (similar to XMLHttpRequest)

### ISSUE THREAD

**Issue by annevk** [w3c/FileAPI#119]
Firefox does this apparently and others do not, but consistency across APIs might make this worth it?

**Comment by angeloko23** [comment]
#77

**Comment by bakulf** [comment]
> Firefox does this apparently and others do not, but consistency across APIs might make this worth it?

This is not correct anymore. I've fixed [bug 1502403](https://bugzilla.mozilla.org/show_bug.cgi?id=1502403).
